frontend/src/scenes/session-recordings/player/SessionRecordingPlayer.tsx

```diff
@@ -198,18 +195,17 @@ export function SessionRecordingPlayer(props: SessionRecordingPlayerProps): JSX.
             'data-attr': 'session-recording-player-view-choice-inspector',
         })
     }
-    if (allowWaterfallView) {
-        viewOptions.push({
-            value: PlaybackViewMode.Waterfall,
-            label: (
-                <div className="space-x-1">
-                    <span>Waterfall</span>
-                    <LemonTag type="success">New</LemonTag>
-                </div>
-            ),
-            'data-attr': 'session-recording-player-view-choice-waterfall',
-        })
-    }
+
+    viewOptions.push({
```

Comment:
I think we may want this inside the `!noInspector` check (and possibly rename that). Unless we want the waterfall view to appear in shared recordings

Reply:
Ah fair... I'll treat it as part of the inspector for now. Probably need to tidy this up down the line though 🙈